### PR TITLE
Display Case Detail Nodesets

### PR DESCRIPTION
**Pull request by wpride**
Extends the CaseListView to allow displaying case lists inside a detail tab

addresses: https://manage.dimagi.com/default.asp?252052

cross-request: https://github.com/dimagi/formplayer/pull/384

screen shot 2017-05-03 at 1 35 00 pm

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js [hunk 1]
```diff
@@ -119,6 +121,28 @@ FormplayerFrontend.module("Menus", function (Menus, FormplayerFrontend, Backbone
         },
 
         getDetailList: function (detailObject) {
+
+            if (detailObject.get('entities')) {
+                var entities = detailObject.get('entities');
+                var listModel = [];
+                // we need to map the details and headers JSON to a list for a Backbone Collection
+                for (var i = 0; i < entities.length; i++) {
+                    var listObj = {};
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js]: nit: these variables get hoisted: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/var#var_hoisting. which is why vars are usually declared at the top

**Review by benrudolph — APPROVED** [review]

**Comment by NoahCarnahan** [comment]
TIL what a dogberg is

**Comment by wpride** [comment]
@NoahCarnahan lollin', Dogberg with the spear outta nowhere! Did you learn what a `Dingus` is too?

**Comment by NoahCarnahan** [comment]
Aw man, Merriam-Webster, New Oxford, and Urban dictionary all disagree on the definition...

**Pull request merged**